Pipfile

```diff
@@ -21,6 +21,7 @@ mypy = "*"
 pre-commit = "*"
 pytest = "*"
 ruff = "*"
+timdex-dataset-api = { git = "git+https://github.com/MITLibraries/timdex-dataset-api.git"}
```

Comment:
Whoops!  Sorry @jonavellecuerdo, missed this the first time around.  Can you move this dependency into the normal dependencies, not `dev`?

Cloudwatch logs from failure when running in StepFunction:
```
  | 2025-01-13T20:35:49.729Z | Traceback (most recent call last):
-- | -- | --
  | 2025-01-13T20:35:49.729Z | File "<string>", line 1, in <module>
  | 2025-01-13T20:35:49.729Z | File "/app/tim/cli.py", line 8, in <module>
  | 2025-01-13T20:35:49.729Z | from timdex_dataset_api import TIMDEXDataset
  | 2025-01-13T20:35:49.729Z | ModuleNotFoundError: No module named 'timdex_dataset_api'
```